components/brave_shields/browser/brave_shields_util.cc

```diff
@@ -332,25 +333,39 @@ void SetFingerprintingControlType(Profile* profile,
   if (!primary_pattern.IsValid())
     return;
 
-  auto* map = HostContentSettingsMapFactory::GetForProfile(profile);
-  map->SetContentSettingCustomScope(
-      primary_pattern, ContentSettingsPattern::Wildcard(),
-      ContentSettingsType::PLUGINS, kFingerprintingV2,
-      GetDefaultAllowFromControlType(type));
+  HostContentSettingsMapFactory::GetForProfile(profile)->
```

Comment:
We do not need two rules for this. The secondary pattern is either "balanced" or it's a wildcard. It should never be both.

Reply:
Fixed to have one fp rule for one url.

Reply:
I'm confused, it's still setting two rules here. We should never be setting both wildcard and balanced for the secondary pattern

Reply:
No, this PR only set one rule for one url but two patterns.
If current www.brave.com has balanced, only one rule (second - `balanced`) is stored.
and if user set other rule(allow), it will have only one rule (second - `*`) is stored.
When new rule is stored previous balanced rule is not deleted automatically because both are different rule. So, clearing step is needed.

Reply:
Ok, I see. I missed the comment